doc/langdef.md

```diff
@@ -116,44 +126,185 @@ In general it is a bad idea for those defining contexts or extensions to use
+such as expression evaluation, rounding mode, or exception handling. However,
+any two not-a-number values will compare equal even if their underlying
+properties are different.
+
+### String and Bytes Values
+
+Strings are sequences of Unicode code points. Bytes are sequences of octets
+(eight-bit data).
+
+Quoted string literals are delimited by either single- or double-quote
+characters, where the closing delimiter must match the opening one, and can
+contain any unescaped character except the delimiter or newlines (either CR or
+LF).
+
+Triple-quoted string literals are delimited by three single-quotes or three
+double-quotes, and may contain any unescaped characters except for the delimiter
+sequence.  Again, the closing delimiter must match the opening one. Triple-quoted
+strings may contain newlines.
+
+Both sorts of strings can include escape sequences, described below.
+
+If preceded by an `r` or `R` character, the string is a _raw_ string and does
+not interpret escape sequences. Raw strings are useful for expressing strings
+which themselves must use escape sequences, such as regular expressions or
+program text.
+
+Bytes literals are represented by string literals preceded by a `b` or `B`
+character. The bytes literal is the sequence of bytes given by the UTF-8
+representation of the string literal. In addition, the octal escape sequence are
+interpreted as octet values rather than as Unicode code points. Both raw and
+multiline string literals can be used for byte literals.
+
+Escape sequences are a backslash (`\`) followed by one of the following:
+
+*   A single-quote(`'`), double-quote(`"`), or backslash(`\`), representing
+    itself.
+*   A code for whitespace:
+    *   `b`: backspace
+    *   `f`: form feed
+    *   `n`: line feed
+    *   `r`: carriage return
+    *   `t`: horizontal tab
+*   A `u` followed by four hexadecimal characters, encoding a Unicode code point
+    in the BMP. Characters in other Unicode planes can be represented with
+    surrogate pairs.
+*   Three octal digits, in the range `000` to `377`. For strings, it denotes the
+    unicode code point. For bytes, it represents an octet value.
+
+Examples:
+
+CEL Literal  | Meaning
+------------ | --------------------------------------------------------------
+`""`         | Empty string
+`'""'`       | String of two double-quote characters
+`'''x''x'''` | String of four characters "x''x"
+`"\""`       | String of one double-quote character
+`"\\"`       | String of one backslash character
+`r"\\"`      | String of two backslash characters
+`b"abc"`     | Byte sequence of 61, 62, 63
+`b"&yuml;"`  | Sequence of bytes 204 and 191 (UTF-8 of &yuml;)
+`b"\303\277"`| Also sequence of bytes 204 and 191
+`"\303\277"` | String of "&Igrave;&iquest;" (code points 204, 191)
+`"\377"`     | String of "&yuml;" (code point 255)
+`b"\377"`    | Sequence of byte 255 (_not_ UTF-8 of &yuml;)
+
+### Aggregate Values
+
+Lists are ordered sequences of values.
```

Comment:
It might be useful to briefly enumerate the aggregate types here. 
They're called out specifically one paragraph at a time below, but the
exact types are pretty clear in other sections, so it would be nice to do
the same here. 

Perhaps, change the section title to `List, Map, and Message Values`?

Reply:
"Numeric Values" is better than "Int, Uint, and Double Values".  I have "String and Bytes Values" and "Booleans and Null" only because I can't think of a good categorical name.  I think I'll leave it as "Aggregate Values".